### PR TITLE
Alert+image+input

### PR DESCRIPTION
**Pull request by vp5h**
Added three new components, Links are as follow
- Input: -https://deploy-preview-4--avidcss.netlify.app/components/input/input
- alert:- https://deploy-preview-4--avidcss.netlify.app/components/alert/alert
- image:- https://deploy-preview-4--avidcss.netlify.app/components/image/image
- fixed routes across component pages

### CODE REVIEW

File: Components/Image/image.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        </nav>
+    </header>
+    <section class="documentation">
+        <aside class="sidebar">
+            <div class="list-topic">
+                <h4 class="highlight">Get Started</h4>
+            </div>
+            <ul>
+                <li><a href="../../getting-started.html">Introduction</a></li>
+                <!-- <li><a href="../Colors/colors.html">Colors</a></li>
```
Comment (Ayushi20-19) [inline Components/Image/image.html:42]: U can remove commented code

**Review by Ayushi20-19 — APPROVED** [review]

File: Components/Alert/alert.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    </header>
+    <section class="documentation">
+        <aside class="sidebar">
+            <div class="list-topic">
+                <h4 class="highlight">Get Started</h4>
+            </div>
+            <ul>
+                <li><a href="../../getting-started.html">Introduction</a></li>
+                <!-- <li><a href="../Colors/colors.html">Colors</a></li>
+          <li><a href="../Typography/typography.html">Typography</a></li> -->
```
Comment (kart-c) [inline Components/Alert/alert.html:43]: remove commented code, if not required

File: Components/Alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+        </main>
+
+    </section>
+</body>
+
+</html>
+
+
+</html>
```
Comment (kart-c) [inline Components/Alert/alert.html]: You have 2 closing tags here

File: Components/Image/image.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+        </main>
+
+    </section>
+</body>
+
+</html>
+
+
+</html>
```
Comment (kart-c) [inline Components/Image/image.html]: 2 closing tags

**Review by kart-c — APPROVED** [review]
Hey Pravesh your code can use some refactor, other than that, it looks good

File: Components/Image/image.html (hunk 4 above)
Comment (vp5h) [reply Components/Image/image.html]: Thanks! fixed in all pages

**Pull request merged**